contrib/Trivy.gitlab-ci.yml

```diff
@@ -0,0 +1,31 @@
+Trivy_container_scanning:
+  stage: test
+  image:
+    name: alpine:3.11
+  variables:
+    # Override the GIT_STRATEGY variable in your `.gitlab-ci.yml` file and set it to `fetch` if you want to provide a `clair-whitelist.yml`
+    # file. See https://docs.gitlab.com/ee/user/application_security/container_scanning/index.html#overriding-the-container-scanning-template
+    # for details
+    GIT_STRATEGY: none
+    IMAGE: "$CI_REGISTRY_IMAGE:$CI_COMMIT_SHA"
+  allow_failure: true
+  before_script:
+    # TODO(tnir): - export TRIVY_VERSION_FOR_TEMPLATE=${TRIVY_VERSION_FOR_TEMPLATE:-v0.4.3}
+    - export TRIVY_VERSION_FOR_TEMPLATE=${TRIVY_VERSION_FOR_TEMPLATE:-fix-template-for-gitlab-integration}
```

Comment:
we could call the trivy binary after fetching and installing.

`TRIVY_VERSION=${TRIVY_VERSION:-$(./bin/trivy --version | sed -e "s/trivy version //")}`

Reply:
@mrueg Thanks for your suggestion. On reflection, we do not have to get version from Trivy binary 😄 

I prefer pinning the Trivy version to make users' DevOps robust as much as possible.